controllers/profiles/common/reconciler.go

```diff
@@ -46,7 +48,12 @@ type StateSupport struct {
 // PerformStatusUpdate updates the SonataFlow Status conditions
 func (s StateSupport) PerformStatusUpdate(ctx context.Context, workflow *operatorapi.SonataFlow) (bool, error) {
 	var err error
+	pl, err := platform.GetActivePlatform(ctx, s.C, workflow.Namespace)
```

Comment:
Don't we already have the platform in the `StateSupport`?

Reply:
hmm... i'm not seeing it
https://github.com/apache/incubator-kie-kogito-serverless-operator/blob/76ef90247c695ea6973e7887c794b03a1541ac63/controllers/profiles/common/reconciler.go#L41-L47

Reply:
I'm wondering if we can reuse the platform we already get in the states/reconciliation. But it's a nit; I think the PR is fine. 👍